### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tylerbryto**
Fixing double FABs bug;
Setting initialRoute in app;
Changin the Navigator pushNamed call.

### CODE REVIEW

File: lib/ui/records_page.dart [hunk 1]
```diff
@@ -8,39 +8,42 @@ class RecordsPage extends StatefulWidget {
...
 }
 
 var container = Container(
   height: 180.0,
   child: Center(
-    child: FloatingActionButton(
-      child: Icon(
-        Icons.keyboard_voice,
-        color: Colors.white,
+    child: CircleAvatar(
```
Comment (odahcam) [inline lib/ui/records_page.dart]: Alternative engineering detected.

File: lib/ui/records_page.dart [hunk 2]
```diff
@@ -8,39 +8,42 @@ class RecordsPage extends StatefulWidget {
...
-    //TODO: fight for the appBar ♥
-    appBar: AppBar(
-      title: Text('Audiozitos',
-      style: TextStyle(color: Colors.white)),
-      centerTitle: true,
-      backgroundColor: Colors.purple,
-    ),
+        //TODO: fight for the appBar ♥
+        appBar: AppBar(
+          title: Text('Audiozitos', style: TextStyle(color: Colors.white)),
```
Comment (odahcam) [inline lib/ui/records_page.dart]: Shouldn't the text be white by default?

File: lib/audiozitos_app.dart [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+class AudiozitosApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) => MaterialApp(
+        title: 'Audiozitos MVP 🏆',
+        theme: ThemeData(
+          primarySwatch: Colors.purple,
+        ),
+        initialRoute: '/',
+        routes: <String, WidgetBuilder>{
+          '/': (context) => RecordsPage(),
```
Comment (odahcam) [inline lib/audiozitos_app.dart]: Curious 🤔

File: lib/ui/records_page.dart (hunk 2 above)
Comment (tylerbryto) [reply lib/ui/records_page.dart]: Yes. My bad ♥

**Comment by odahcam** [comment]
OOooooooooorrra build no OSX passando já, que delícia.

**Pull request merged**